content/en/docs/contribute/localization.md

```diff
@@ -277,7 +277,7 @@ SIG Docs welcomes upstream contributions and corrections to the English source.
 
 ## Help an existing localization
 
-You can also help add or improve content to an existing localization. Join the [Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/) for the localization, and start opening PRs to help.
+You can also help add or improve content to an existing localization. Join the [Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/) for the localization, and start opening PRs to help. Please limit pull requests to a single localization since pull requests that elapse multiple localizations could be difficult to review.
```

Comment:
```suggestion
You can also help add or improve content to an existing localization. Join the [Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/) for the localization, and start opening PRs to help. Please limit pull requests to a single localization since pull requests that change content in multiple localizations could be difficult to review.
```

Reply:
Hi @sftim,

Thanks for reviewing the pull request. I am okay with the suggested change but, unfortunately, I am not familiar with amending and force-push workflow.  Please let me know the next steps and I would follow along to submit these changes.

Reply:
If you accept the suggestion (there's a “commit suggestion”) then I can add a label to tell the automation (“Tide”) to squash that into your first commit when this merges.

So all you need to do is to find the “Commit suggestion” button on this PR and click it. See the [Files changed](https://github.com/kubernetes/website/pull/20431/files) tab.

Reply:
@sftim This should be all set from my end. I guess we need to add the tide label now.